src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");

```

Comment:
The message should indicate what type of `Machine` is working. It's better to specify that it is a `Bulldozer` to make the message more informative as per the checklist.